packages/ckeditor5-dev-env/lib/release-tools/utils/transformcommitforckeditor5devpackage.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const transformCommitForCKEditor5Package = require( './transformcommitforckeditor5package' );
+const getChangedFilesForCommit = require( './getchangedfilesforcommit' );
+
+/**
+ * Parses a single commit for `ckeditor5-dev` package.
+ *
+ * @param {Commit} commit
+ * @param {Object} {context}
+ * @param {Boolean} context.displayLogs Whether to display the logs.
+ * @param {Object} context.packageData Content from the 'package.json' for given package.
+ * @returns {Commit}
+ */
+module.exports = function transformCommitForCKEditor5DevPackage( commit, context ) {
```

Comment:
Isn't there more gitty way of retrieveing such commits? Like in http://stackoverflow.com/questions/16343659/how-to-show-git-log-history-for-a-sub-directory-of-a-git-repo?